src/nls/fi/strings.js

```diff
@@ -624,7 +624,25 @@ define({
     "CMD_TOGGLE_RECENT_PROJECTS"                : "Viimeisimmät projektit",
 
     // extensions/default/WebPlatformDocs
-    "DOCS_MORE_LINK"                            : "Lue lisää"
+    "DOCS_MORE_LINK"                            : "Lue lisää",
+
+    //extensions/default/CodeFolding
+    "COLLAPSE_ALL"                  : "Pienennä kaikki",
```

Comment:
@abose 5 of the translations are for the menu items under `View` i.e., 'collapse all', 'expand all', 'collapse current', 'expand current', 'collapse custom regions'. I think it is good idea to keep these menu items (as they also show the shortcut keys). If there are strong reasons why you think we should remove them then we can discuss them.

As for the other preference strings, you are absolutely right. I've only kept them to use in the options parameter (name and description) when defining preferences. So they can be taken out of the non root strings.